src/structures/interfaces/Application.js

```diff
@@ -23,23 +23,29 @@ class Application extends Base {
      */
     this.id = data.id;
 
-    /**
-     * The name of the app
-     * @type {string}
-     */
-    this.name = data.name;
-
-    /**
-     * The app's description
-     * @type {string}
-     */
-    this.description = data.description;
-
-    /**
-     * The app's icon hash
-     * @type {string}
-     */
-    this.icon = data.icon;
+    if ('name' in data) {
+      /**
+       * The name of the app
+       * @type {string}
```

Comment:
All of these are now `?string`

Reply:
Doesn't `?type` only mean `type|null`? In this case the properties are optional, not nullable